src/GLib/MutableTypes.jl

```diff
@@ -29,7 +29,7 @@ mutable{T <: Ptr, N}(x::Array{T, N}, i = 1) = mutable(x[i])
 mutable{T}(x::Ptr{T}, i = 1) = x + (i - 1) * sizeof(T)
 mutable{T}(x::T, i) = (i == 1 ? mutable(x) : error("Object only has one element"))
 
-_addrof{T}(b::T) = ccall(:jl_value_ptr, Ptr{T}, (Ptr{Any},), &b)
+_addrof{T}(b::T) = pointer_from_objref(b)
```

Comment:
@vtjnash is this correct ?

Reply:
The iters error seems to be caused by invalid GtkTreeIter when doing this:

```
ls=ListStore(Int32,Bool)
push!(ls,(44,true))
push!(ls,(33,true))
insert!(ls, 2, (35, false))
```

insert! gets a valid iter from here, but it seems to become invalid at the insert time. I'm not sure what's going on.

https://github.com/JuliaGraphics/Gtk.jl/blob/master/src/lists.jl#L106

Reply:
Yes